synapse/lib/storm.py

```diff
@@ -945,6 +945,7 @@ def __init__(self, snap, opts=None, user=None):
 
         self.opts = opts
         self.snap = snap
+        self.view = snap.view
```

Comment:
This feels like we're leaking an abstraction that we were trying to move away from